synonymdev/bitkit-react-native
# Update env Templates

### Description

- Adds empty string to `TREASURE_HUNT_HOST` in env templates to prevent "`TREASURE_HUNT_HOST` is not defined" errors when git cloning and installing.
- Not sure if there's a better way to handle this?

### Type of change

### Tests

### Screenshot / Video
![Simulator Screenshot - iPhone 14 - 2024-02-01 at 14 53 25](https://github.com/synonymdev/bitkit/assets/8532651/96f89836-6cc7-45c7-8245-abbe449cf66f)

### QA Notes
- We should now be able to run the following command without an error resembling the attached screenshot.
```
git clone git@github.com:synonymdev/bitkit.git && cd bitkit && git checkout update-env-templates && yarn install && cd ios && pod install && cd ../ && yarn ios
```
